apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/utils/EndpointJsonResponseComparatorIT.java

```diff
@@ -929,6 +932,43 @@ public static String eobReadCarrierWithTaxNumbers() {
     return jsonInterceptor.getResponse();
   }
 
+  /**
+   * @return the results of the {@link
+   *     ExplanationOfBenefitResourceProvider#read(org.hl7.fhir.dstu3.model.IdType)} operation for
+   *     Carrier claims, with the {@link
+   *     ExplanationOfBenefitResourceProvider#HEADER_NAME_INCLUDE_TAX_NUMBERS} set to <code>false
+   *     </code>
+   */
```

Comment:
Missing doc description (Also I think its the wrong doc?)

Reply:
done